app/models/hearing.rb

```diff
@@ -51,6 +57,10 @@ def defendants
     prosecution_cases.flat_map { |prosecution_case| prosecution_case['defendants'] }
   end
 
+  def defence_details
+    hearing_body['defenceCounsels']
```

Comment:
As with prosecutionCaseCounsels, we might need to consider a safe navigation via `dig` or similar.
https://github.com/ministryofjustice/laa-court-data-adaptor/blob/56292ca03065155b3325cae7c2163f325a87b554/app/models/hearing.rb#L35-L37